specshaper/encrypt-bundle/3.0/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "SpecShaper\\EncryptBundle\\SpecShaperEncryptBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "ENCRYPT_KEY": "change_me"
```

Comment:
I’d make the name of the env var a bit more specific - like SPEC_SHARPER_ENCRYPT_KEY

Reply:
Good suggestion. Done.